generators/client-2/templates/src/main/webapp/app/layouts/navbar/navbar.html

```diff
@@ -95,91 +93,71 @@
-                            <a ui-sref="<%=jhiPrefix%>-tracker" ng-click="vm.collapseNavbar()">
-                                <span class="glyphicon glyphicon-eye-open"></span>&nbsp;
-                                <span translate="global.menu.admin.tracker">User tracker</span>
-                            </a>
-                        </li>
-                        <%_ } _%>
                         <li ui-sref-active="active">
-                            <a ui-sref="<%=jhiPrefix%>-metrics" ng-click="vm.collapseNavbar()">
+                            <a ui-sref="jhi-metrics" (click)="collapseNavbar()">
                                 <span class="glyphicon glyphicon-dashboard"></span>&nbsp;
                                 <span translate="global.menu.admin.metrics">Metrics</span>
                             </a>
                         </li>
                         <li ui-sref-active="active">
-                            <a ui-sref="<%=jhiPrefix%>-health" ng-click="vm.collapseNavbar()">
+                            <a ui-sref="jhi-health" (click)="collapseNavbar()">
                                 <span class="glyphicon glyphicon-heart"></span>&nbsp;
                                 <span translate="global.menu.admin.health">Health</span>
                             </a>
                         </li>
                         <li ui-sref-active="active">
-                            <a ui-sref="<%=jhiPrefix%>-configuration" ng-click="vm.collapseNavbar()">
+                            <a ui-sref="jhi-configuration" (click)="collapseNavbar()">
                                 <span class="glyphicon glyphicon-list-alt"></span>&nbsp;
                                 <span translate="global.menu.admin.configuration">Configuration</span>
                             </a>
                         </li>
                         <li ui-sref-active="active">
-                            <a ui-sref="audits" ng-click="vm.collapseNavbar()">
+                            <a ui-sref="audits" (click)="collapseNavbar()">
                                 <span class="glyphicon glyphicon-bell"></span>&nbsp;
                                 <span translate="global.menu.admin.audits">Audits</span>
                             </a>
                         </li>
                         <li ui-sref-active="active">
-                            <a ui-sref="logs" ng-click="vm.collapseNavbar()">
+                            <a ui-sref="logs" (click)="collapseNavbar()">
                                 <span class="glyphicon glyphicon-tasks"></span>&nbsp;
                                 <span translate="global.menu.admin.logs">Logs</span>
                             </a>
                         </li>
-                        <li ng-show="vm.swaggerEnabled" ui-sref-active="active">
-                            <a ui-sref="docs" ng-click="vm.collapseNavbar()">
+                        <li [hidden]="!swaggerEnabled" ui-sref-active="active">
```

Comment:
`ngIf` and `[hidden]` does not do the same thing:
`ngIf` remove the element from the DOM while hidden just add a CSS property to hide the element, that's what `ng-show` does, that's why I put `hidden` and not `ngIf` there, is that ok for you ? :)